src/index.js

```diff
@@ -118,7 +118,8 @@ class Service {
   raw (method, params) {
     if(typeof method === 'undefined') {
       return Promise
-        .reject(new Error('params.method must be defined.'));
+        .reject(new Error('params.method must be defined.'))
+        .catch(errorHandler);
     }
```

Comment:
For the error handling, since we return a promise back, so we handle them uniformly as catch them at the end of the return. Is this OK?